english/lovecraftian/shoggoth.md

```diff
@@ -6,3 +6,5 @@ This really hurts.
 But just then you:
 
 [See an ostrich?] (../hospital_bed/hospital_bed.md)
+
+[Short Lovecraftian recap of events leading to this point]  (../london/dr_lawrence_crane.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.